ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.collection.view.js

```diff
@@ -36,6 +36,10 @@ module.exports = Marionette.CollectionView.extend({
     if (this.options.hideNewForm) {
       return child.get('type') !== 'new-form'
     }
-    return child.get('type') !== 'basic' && child.get('type') !== 'text'
+    return (
+      child.get('type') !== 'basic' &&
+      child.get('type') !== 'text' &&
+      child.get('createdBy') !== 'system'
```

Comment:
❗️ 	If we don't show the system templates here, the users won't be able to select a system template as their default. We removed the three dot dropdown menu from the search form tiles when they select an existing form so now this is the only place they can update their defaults.